python-cim/samples/dump_class_instance.py

```diff
@@ -31,9 +41,9 @@ def compute_instance_hash(index, instance):
     return ""
 
 
-def main(type_, path, namespaceName, className, key_specifier=None):
-    if type_ not in ("xp", "win7"):
-        raise RuntimeError("Invalid mapping type: {:s}".format(type_))
+def main(type_, path, namespaceName, className, key_specifier=None, csv=None):
+    #if type_ not in ("xp", "win7"):
+    #    raise RuntimeError("Invalid mapping type: {:s}".format(type_))
```

Comment:
why is this commented out? 

if there's a good reason, we should remove the code completely, since git tracks the history.